linux/shim/init_state.go

```diff
@@ -345,10 +346,7 @@ func (s *stoppedState) Delete(ctx context.Context) error {
 }
 
 func (s *stoppedState) Kill(ctx context.Context, sig uint32, all bool) error {
-	s.p.mu.Lock()
-	defer s.p.mu.Unlock()
-
-	return s.p.kill(ctx, sig, all)
+	return errdefs.ToGRPCf(errdefs.ErrNotFound, "process %s not found", s.p.id)
```

Comment:
maybe use failed preconditions here? Although I guess not found make it easier, only need to check against one type of error

Reply:
Its this way because we map no such process syscall error to NotFound